test/unit/txIdFromHeader.test.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: loopback-next-hello-extension
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect} from '@loopback/testlab';
+import {get, getControllerSpec} from '@loopback/core';
+import {txIdFromHeader} from '../..';
+
+describe('@txHeaderFromId', () => {
```

Comment:
Please move this file to `test/unit/decorators/txIdFromHeader.test.ts`. Considering that this is a unit-test that should be executing only a single class (single source file), it may be better to include ".decorator" type in the file name too, e.g. `test/unit/decorators/txIdFromHeader.decorator.test.ts`.